soweego/importer/base_dump_extractor.py

```diff
@@ -33,3 +37,23 @@ def get_dump_download_urls(self) -> Iterable[str]:
         :rtype: str
         """
         raise NotImplementedError
+
+    def _commit_entity(self, db_manager, entity, delay: int = 10):
+        success = True
+        session = db_manager.new_session()
+        try:
+            session.add(entity)
+            session.commit()
+        except Exception as ex:
+            LOGGER.error("Failed to commit % s due to %s" % (entity, ex))
```

Comment:
```suggestion
            LOGGER.error('Failed to commit %s due to %s', entity, ex)
```
2 minor things related to style:
1. remember to use simple quotes `'`;
2. log message variables should be passed as arguments to the function call.